app/controllers/hub/organizations_controller.rb

```diff
@@ -2,27 +2,13 @@ module Hub
   class OrganizationsController < ApplicationController
     include AccessControllable
     before_action :require_sign_in
-    load_and_authorize_resource :organization, parent: false
+
     before_action :load_coalitions
+    load_and_authorize_resource :organization, parent: false
```

Comment:
@caitbot FYI:

It seems peaceful to use the less-verbose `load_and_authorize_resource`. The reason it works is that the controller is the OrganizationsController so cancancan will know to query for `Organization` model instances. (It wasn't written this way originally because, before you refactored VitaPartner to have the two subclasses, it wasn't possible to query for `Organization.where(...)`.)

For more info about `parent`, see this sentence: "If a name is provided which does not match the controller it assumes it is a parent resource. Child resources can then be loaded through it." and the full docs for `load_resource`: https://www.rubydoc.info/github/CanCanCommunity/cancancan/CanCan%2FControllerAdditions%2FClassMethods:load_resource